src/test/java/com/wikia/webdriver/testcases/discussions/CreatingPostTests.java

```diff
@@ -291,4 +325,46 @@ private void assertThatUserCanClickPostAndGoToPostDetailsPage() {
     Assertion.assertTrue(PostDetailsPage.is(url));
     Assertion.assertTrue(url.endsWith(postDetailsUrl));
   }
+
+  private PostEntity.Data createNaughtyStringsPostRemotely(User user) {
+    String description = FIRST_LINE + ContentLoader.loadWikiTextContent("blns.txt");
+    String title = "Naughty strings";
+    return DiscussionsOperations.using(user, driver).createCustomPost(siteId, title, description);
+  }
+
+  private PostEntity.Data createWeirdCharactersPostOnWiki(String wiki) {
+    setUp(wiki);
+    return createNaughtyStringsPostRemotely(User.USER);
+  }
+
+  private void cleanUp(PostEntity.Data post) {
+    DiscussionsOperations.using(User.STAFF, driver).deletePost(post, siteId);
+  }
+
+  private void assertPostWithWeirdCharactersDisplayedOnPostsListPage(String wiki) {
+    PostEntity.Data post = createWeirdCharactersPostOnWiki(wiki);
+    String description = new PostsListPage()
+      .open()
+      .getPost()
+      .findPostById(post.getId())
+      .findDescription();
+    try {
+      Assertion.assertStringContains(description, FIRST_LINE);
+    } finally {
+      cleanUp(post);
+    }
+  }
+
+  private void assertPostWithWeirdCharactersDisplayedOnPostDetailsPage(String wiki) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "assertPostWithWeirdCharactersDisplayedOnPostDetailsPage" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)